kadena-beta/docs/Kadena-README.md

```diff
@@ -1,10 +1,40 @@
-# Kadena Blockchain Documentation
+# Kuro Blockchain Documentation
 ---
 
-Kadena Version: 1.1.x
+Kuro Version: 1.1.x
 
 # Change Log
 
+* Version 1.4
```

Comment:
Why are we duplicating the changelog here?

Reply:
Previous versions of the document had the changelog. It might have been since `kadena-beta` was bundled to clients and this README would be their initial introduction. But we also included the Changelog according to `scripts/build-beta-distro.sh`. Should I just delete this entire section?

Reply:
Yeah